chia/consensus/block_body_validation.py

```diff
@@ -153,24 +162,25 @@ async def validate_block_body(
 
     # We check in header validation that timestamp is not more that 10 minutes into the future
     if (
-        block.foliage_transaction_block is not None
```

Comment:
we know `block.foliage_transaction_block` is not `None` because we've already checked that this is a transaction block. right?

Reply:
yes exactly